README.md

```diff
@@ -8,6 +8,34 @@
 [![Chat](https://img.shields.io/matrix/cylc-general:matrix.org)](https://matrix.to/#/#cylc-general:matrix.org)
 
 
+### Installation
+
+It is recommended to install as a Vim package
+(see [`:help packages`](https://vimhelp.org/repeat.txt.html#packages)):
+
+```bash
+mkdir -p ~/.vim/pack/vendor/start
+cd ~/.vim/pack/vendor/start
+git clone https://github.com/cylc/cylc.vim.git
+```
+
+Otherwise, use your favourite plugin manager, such as
+[pathogen.vim](https://github.com/tpope/vim-pathogen).
```

Comment:
Pathogen seems a bit dead looking (note on the project saying not to use it).

Possibly replace with vim-plug:

```suggestion
Otherwise, use your favourite plugin manager, such as
[vim-plug](https://github.com/junegunn/vim-plug):

`Plug 'cylc/cylc.vim'`
```

Reply:
The note on pathogen is not because it's dead per se, it's because its behaviour is so close to vim 8's package management that you might as well use that. That's what I like it for: it backports that functionality. Could still make more sense to mention vim-plug than that, not sure...